numpy/core/tests/test_dtype.py

```diff
@@ -1355,10 +1355,6 @@ class mytype:
         # unnecessary restriction, but one that has been around forever:
         assert np.dtype(mytype) == np.dtype("O")
 
-        with pytest.raises(RuntimeError):
-            # Registering a second time should fail
-            create_custom_field_dtype(blueprint, mytype, 0)
```

Comment:
The test only made sense because I mapped the `pytype -> dtype`, but for legacy user-dtypes where the pytype does not subclass `np.generic` we actually do not do that, so testing for this error was actually incorrect.